core/storage/user/gae_models.py

```diff
@@ -67,6 +67,20 @@ def get_by_normalized_username(cls, normalized_username):
             cls.normalized_username == normalized_username).get()
 
 
+class UserContributionsModel(base_models.BaseModel):
+    """Tracks statistics for a particular user.
+    Includes subsequently deleted and private explorations.
+
+    Instances of this class are keyed by the user id.
+    """
+    # IDs of explorations that this user has created.
+    created_explorations = ndb.TextProperty(repeated=True, 
+        indexed=True, default=None)
+    # IDs of explorations that this user has created.
```

Comment:
IDs of explorations that this user has made a positive (i.e. non-revert) commit to. This includes private explorations and explorations that subsequently get deleted.